experimental/devices/epd/doc.go

```diff
@@ -0,0 +1,16 @@
+// Copyright 2018 The Periph Authors. All rights reserved.
+// Use of this source code is governed under the Apache License, Version 2.0
+// that can be found in the LICENSE file.
+
+// Package epd controls Waveshare e-paper series displays
+//
+// More details
+//
+// See https://periph.io/device/epd/ for more details about the device.
+//
+// Based on https://github.com/soonuse/epd-library-python
+//
+// Product page:
+// https://www.waveshare.com/wiki/2.13inch_e-Paper_HAT
+// https://www.waveshare.com/wiki/1.54inch_e-Paper_Module
```

Comment:
What I find more useful is the datasheet; https://www.waveshare.com/w/upload/e/e6/2.13inch_e-Paper_Datasheet.pdf

For example, you can implement temperature reading (see page 9).